doc/users.md

```diff
@@ -0,0 +1,29 @@
+[doc](../README.md) > Users
+
+# Users Management
+
+Users information has been introduced in [schema update 07](../server/db/satnogs-07.psql). There's a new table that holds
```

Comment:
It is ```satnogs-08.psql``` file.

Reply:
Good catch! It was 07 when I was developing it, but your code (with your -07) got to master first.